app/javascript/dashboard/api/attributes.js

```diff
@@ -1,9 +1,14 @@
+/* global axios */
 import ApiClient from './ApiClient';
 
 class AttributeAPI extends ApiClient {
   constructor() {
     super('custom_attribute_definitions', { accountScoped: true });
   }
+
+  getAttributeModel(modelId) {
```

Comment:
Unused API method

Reply:
Now it is using, by direct calling `get()`, it is not showing contact attributes.